index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+ <html lang="en">
```

Comment:
Could use add some comments in HTML to section out sections, for example 'left-bar starts' code 'left-bar ends' - easier for you to also make changes if you're trying to find a section.